release_bot/cli.py

```diff
@@ -21,27 +21,45 @@
 
 
 class CLI:
+
     @staticmethod
     def parse_arguments():
         """Parse application arguments"""
+
+        def run_init(_):
+            """
+            Makes the bot to run the init script when the `release_bot init`
+            """
+            configuration.init = True
+
+        def run_bot(args):
+            """
+            Makes the bot to perform normal release tasks
+            """
+            if args.configuration:
+                args.configuration = Path(args.configuration).resolve()
+                if not args.configuration.is_file():
+                    configuration.logger.error(
+                        f"Supplied configuration file is not found: {args.configuration}")
+                    exit(1)
+
+                    if args.debug:
+                        configuration.logger.setLevel(logging.DEBUG)
+                        for key, value in vars(args).items():
+                            setattr(configuration, key, value)
+
         parser = argparse.ArgumentParser(description="Automatic releases bot", prog='release-bot')
         parser.add_argument("-d", "--debug", help="turn on debugging output",
                             action="store_true", default=False)
         parser.add_argument("-c", "--configuration", help="use custom YAML configuration",
                             default='')
         parser.add_argument("-v", "--version", help="display program version", action='version',
                             version=f"%(prog)s {configuration.version}")
+        parser.set_defaults(func=run_bot)
 
-        args = parser.parse_args()
+        subparsers = parser.add_subparsers()
+        parser_init = subparsers.add_parser('init')
```

Comment:
Can you add some help string? ... `subparsers.add_parser('init', help='Initialize boo bar baz')`